.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -23,6 +23,7 @@ assignees: ''
   - [ ] All tests pass in the ["GMT Dev Tests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
   - [ ] All tests pass in the ["Doctests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_doctests.yaml)
   - [ ] Deprecations and related tests are removed for this version by running `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository
+- [ ] Update warning in `pygmt.show_versions()` and note in [Common installation issues](https://www.pygmt.org/dev/install.html#common-installation-issues) regarding GMT-ghostscript incompatibility
```

Comment:
```suggestion
- [ ] Update warnings in `pygmt.show_versions()` and notes in [Common installation issues](https://www.pygmt.org/dev/install.html#not-working-transparency) regarding GMT-Ghostscript incompatibility
```

Reply:
See commits https://github.com/GenericMappingTools/pygmt/pull/3249/commits/78eadd8ec1e28c849e781843776879b5dabb09da and https://github.com/GenericMappingTools/pygmt/pull/3249/commits/299ec6b553a34b175320b2383e98d9196afc99b0.